proto/panacea/did/v3/did.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package panacea.did.v3;
+
+option java_multiple_files = true;
+option go_package = "github.com/medibloc/panacea-core/x/did/types";
+
+import "gogoproto/gogo.proto";
+
+// DIDDocument defines a W3C DID Document with document data type to specify the version of document spec.
+message DIDDocument {
+  bytes document = 1;
+  string document_data_type = 2;
+}
```

Comment:
```suggestion
message DIDDocument {
  bytes data = 1;
  string type = 2;
}
```
Let's go with simple names, as https://github.com/libp2p/go-libp2p/blob/5741b6c9bbcc1185bdf94d816dca966b37ce61ff/core/crypto/pb/crypto.proto#L14

Reply:
```suggestion
message DIDDocument {
  bytes document = 1;
  string document_type = 2;
}
```
Sorry. Let's go like this. This would be clearer since you're adding a `deactivated` field as https://github.com/medibloc/panacea-core/pull/633/files#diff-e50f8f7e53eb90e2acaa0285df776b2eb2b13f0dd5a28ab17a0c021224f509fcR13

Reply:
I will reflect this when we reopen this PR in the future.